src/System.Runtime/tests/System/DateTimeTests.cs

```diff
@@ -1038,6 +1038,69 @@ public static IEnumerable<object[]> ParseExact_ValidInput_Succeeds_MemberData()
             yield return new object[] { "1234-05-06T07:00:00Z", "yyyy-MM-dd'T'HH:mm:ssFFFZ", CultureInfo.InvariantCulture, DateTimeStyles.None, TimeZoneInfo.ConvertTimeFromUtc(new DateTime(1234, 5, 6, 7, 0, 0, DateTimeKind.Utc), TimeZoneInfo.Local) };
             yield return new object[] { "1234-05-06T07:00:00GMT", "yyyy-MM-dd'T'HH:mm:ssFFFZ", CultureInfo.InvariantCulture, DateTimeStyles.None, TimeZoneInfo.ConvertTimeFromUtc(new DateTime(1234, 5, 6, 7, 0, 0, DateTimeKind.Utc), TimeZoneInfo.Local) };
 
+
+            yield return new object[] { "9", "\"  \"%d", CultureInfo.InvariantCulture, DateTimeStyles.AllowLeadingWhite, new DateTime(DateTime.Now.Year, 1, 9, 0, 0, 0) };
+            yield return new object[] { "15", "\' \'dd", CultureInfo.InvariantCulture, DateTimeStyles.AllowLeadingWhite, new DateTime(DateTime.Now.Year, 1, 15, 0, 0, 0) };
```

Comment:
Maybe I'm misunderstanding, but doesn't AllowLeadingWhitespace primarily apply to ignoring whitespace in the input string, not the pattern?

Reply:
That isn't what the implementation suggests. Quoted white space within the format string is allowed. Although, I couldn't really find good documentation on it to show it is supported. MSDN only talks about the input string.

Reply:
https://msdn.microsoft.com/en-us/library/ms131038(v=vs.110).aspx

> AllowLeadingWhite - Specifies that white space not defined by format can appear at the beginning of s.

https://referencesource.microsoft.com/#mscorlib/system/globalization/datetimeparse.cs,4038

Reply:
> Specifies that white space not defined by format can appear at the beginning of s.

Right... that's not what your tests are testing... the input strings you added have no whitespace... what am I missing?

Reply:
You wanted me to add tests to cover all the code paths in RemoveLeadingInQuoteSpaces() / RemoveTrailingInQuoteSpaces(). Those methods are only ever called when removing white space from the format string (not from the input string). I understand that the tests I added do not verify the correctness of the whitespace in the input string (and we should definitely add those).

If there is a bug in the implementation of TryParseExact, we should address that, but I am not sure what the expected behavior should be based on the docs.

Reply:
> Those methods are only ever called when removing white space from the format string (not from the input string).

Ah, that's what I was missing, thanks.